docs/2021/ui/updates/shruti/2021-07-12.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Week 6
+author: Shruti Agarwal
+---
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2021 Shruti Agarwal <mail2shruti.ag@gmail.com>
+-->
+
+* Added the candidate license page.
+* Added the prototype for add candidate license page in Admin.
+* Created the sample page for select license.
+* Added the downloading of License CSV in admin.
+* Fixed the styling textarea in dark mode.
+* Created the widget index.js file where created the imports of all the widgets we required in the codebase.
+* Refactor the widgets imports it will help us to not impot the widget again and again.
+* Fixed the license headers in all the files to maintain the symmery.
+* Added the comments in all the files of services, apis, components, pages, data and many more.
+* Added the comments for all the imports and state variables.
+* Fixed the tokens in all the apis.
+* On clicking the close button of snackbars in the `deleteUploads` and `unlinkFolder` it is breaking replaced the onClose function to setShow.
+* Added the small pop up for loading while changing the pagination or entries, enhances the UX and closes [#91](https://github.com/fossology/FOSSologyUI/issues/91)
+* Added the `jsconfig.json` for the relative paths.
+* modified the `.eslintrc.json` fro resolving the imports.
+* Fixed all the imports to relative paths.
+
+
+## Screenshots
+### Candidate License
+<p align="center">
+   <img src="https://user-images.githubusercontent.com/56133783/124560675-b4ff8680-de5a-11eb-8a2c-c3badacb4f49.png" alt="candidate-license"/>
```

Comment:
Please move images to the static/img/reactUI/ folder itself so we do not loose them.

Reply:
Ohkk then I will add all the images because in previous reports also I have added the links.